test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -126,7 +127,13 @@ var _ = Describe("[rfe_id:27368][performance]", Ordered, func() {
 				Expect(err).ToNot(HaveOccurred(), "Failed to get the Tuned profile for node %s", node.Name)
 				degradedCondition := findCondition(tunedProfile.Status.Conditions, "Degraded")
 				Expect(degradedCondition).ToNot(BeNil(), "Degraded condition not found in Tuned profile status")
-				Expect(degradedCondition.Status).To(Equal(corev1.ConditionFalse), "Tuned profile is degraded")
+				isNodeBasedOnVM, err := infrastructure.IsVM(&node)
+				Expect(err).ToNot(HaveOccurred(), "Failed to detect if the node is based on VM")
+				if isNodeBasedOnVM {
+					testlog.Warning(fmt.Sprintf("Tuned profile is degraded. A warning raised as the node is based on a VM, and is expected. Error message: %s", degradedCondition.Message))
```

Comment:
not all failures here are  expected,  please remove "and is expected"

Reply:
Done